src/components/Client/index.js

```diff
@@ -46,6 +68,13 @@ class Client extends Component {
     // The server has updated us with a list of all users currently on the system
     onUpdateClient = message => {
 
+        /**
+         * Style Feedback: Some teams favor never using `let` and only using `const` as declaring a variable
```

Comment:
Yes, I'm moving that direction as well, I just lapse occasionally. Thanks for pointing it out.